src/tests/integration/test_postgreSQL.py

```diff
@@ -0,0 +1,17 @@
+# flake8: noqa
+from fixtures.database import *
+
+
+def test_meta_cmd_display(ip_with_postgreSQL):
+    out = ip_with_postgreSQL.run_cell("%sql \d")
```

Comment:
run_cell wont raise any errors if it fails, but it will set a property (I forgot the name) that will contain the error if any, let's add an assert to ensure that no error was raised

Reply:
I guess you are referring to the [error_in_exec](https://github.com/ipython/ipython/blob/main/IPython/core/interactiveshell.py#L3022) from `IPython/core/interactiveshell.py`

Reply:
However, I am observing some errors from ip fixture in `conftest.py`
Issue: Two `sqlite3.OperationalError` are thrown

### Way to reproduce: 

1. Test case code:
```
def test_meta_cmd_display(ip_with_postgreSQL):
    print ("Running test_meta_cmd_display\n")
    out = ip_with_postgreSQL.run_cell("%sql \d")  # noqa: W605
    assert len(out.result) > 0
    assert ("public", "taxi", "table", "ploomber_app") in out.result
    print ("Ending test_meta_cmd_display\n")
```

2. Run with `pytest src/tests/integration/test_postgreSQL.py -s -k "test_meta_cmd_display"  `

3. Showing on terminal
```
Running test_meta_cmd_display

*  (postgreSQLTest) postgresql://ploomber_app:***@localhost:5432/db
1 rows affected.
Out[1]: [('public', 'taxi', 'table', 'ploomber_app')]
Ending test_meta_cmd_display

.(sqlite3.OperationalError) no such table: test
[SQL: DROP TABLE test]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
(sqlite3.OperationalError) no such table: author
[SQL: DROP TABLE author]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
```

### Root cause

It's from ip fixture in `conftest.py`
https://github.com/ploomber/jupysql/blob/master/src/tests/conftest.py#L44

It will trying to init. the `InteractiveShell` instance and register a few magics, then create some fake tables into sqlite engine(test, author) then yield the ip, and finally drop those fake tables.

However, in our case in every ip_with_{database_name}: we will deselect the SQLite engine and create our own test data, of course it doesn't contain those fake tables.

### Root cause - Solution Proposal

We can create the new `ip_wo_engine` in `src/tests/conftest.py`

Then every ip_with_{database_name} in `src/tests/conftest.py` should inherit our new `ip_wo_engine`

In `src/tests/conftest.py`
```
@pytest.fixture
def ip_wo_engine():
    """Provides an IPython session in which tables have been created"""
    ip_session = InteractiveShell()
    ip_session.register_magics(SqlMagic)
    ip_session.register_magics(RenderMagic)
    ip_session.register_magics(SqlPlotMagic)
    ip_session.register_magics(SqlCmdMagic)
    yield ip_session
```

Reply:
commented in the review on this